src/graph.rs

```diff
@@ -348,27 +348,30 @@ impl Eq for ResolutionError {}
 impl fmt::Display for ResolutionError {
   fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
     match self {
-      Self::InvalidDowngrade { specifier, .. } => write!(f, "Modules imported via https are not allowed to import http modules.\n  Importing: {}", specifier),
-      Self::InvalidLocalImport { specifier, .. } => write!(f, "Remote modules are not allowed to import local modules. Consider using a dynamic import instead.\n  Importing: {}", specifier),
+      Self::InvalidDowngrade { specifier, .. } => write!(f, "Modules imported via https are not allowed to import http modules.\n  Importing: {specifier}"),
+      Self::InvalidLocalImport { specifier, .. } => write!(f, "Remote modules are not allowed to import local modules. Consider using a dynamic import instead.\n  Importing: {specifier}"),
       Self::ResolverError { error, .. } => error.fmt(f),
       Self::InvalidSpecifier { error, .. } => error.fmt(f),
     }
   }
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum Resolved {
+pub struct ResolutionResolved {
+  /// Specifier to.
+  pub specifier: ModuleSpecifier,
+  /// Referrer range.
+  pub range: Range,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Resolution {
   None,
-  Ok {
-    /// Specifier to.
-    specifier: ModuleSpecifier,
-    /// Referrer range.
-    range: Box<Range>,
```

Comment:
This clippy suggestion to add a `Box` here seems like it has a limitation analyzing what's in `ResolutionError`, which was much bigger. There will be a lot of `Resolution::None` in the graph so making these almost the same size seems worthwhile? I put the actual variants in boxes.